lib/saluki-components/src/sources/checks/builder/python/python_modules.rs

```diff
@@ -351,6 +351,7 @@ mod tests {
                         panic!("Unexpected event title: {}", ev.title());
                     }
                 }
+                saluki_core::data_model::event::Event::Log(_) => todo!(),
```

Comment:
`saluki_core::data_model::event::Event` is already in scope as `Event` here, so this can just be `Event::Log(_) => todo!()`.